### DISCUSSION TITLE
Windows Clients do login but app / browser extension closes instantly, access to passwords given (Current version / see video)

### DISCUSSION THREAD

**Discussion by fthobe** [dani-garcia/vaultwarden#4824] [Q&A]
### Screenrecording of Issue
![ScreenRecording2024-08-06at16 17 59-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/0babea48-6a93-49a8-89c8-84f15012ea1a)
https://github.com/user-attachments/assets/d2260f72-05b0-47f2-b020-2bb696cc3da5

### Subject of the issue
The applications / extensions work flawlessly on MacOS / iOS / Android, but immediatly close on Windows (Chrome extension / App alike)

### Your environment (Generated via diagnostics page)
* Vaultwarden version: v1.31.0
* Web-vault version: v2024.5.1b
* OS/Arch: linux/x86_64
* Running within a container: true (Base: Alpine)
* Environment settings overridden: true
* Uses a reverse proxy: true
* IP Header check: true (X-Real-IP)
* Internet access: true
* Internet access via a proxy: false
* DNS Check: true
* Browser/Server Time Check: true
* Server/NTP Time Check: true
* Domain Configuration Check: true
* HTTPS Check: true
* Database type: SQLite
* Database version: 3.46.0
* Clients used: 
* Reverse proxy and version: 
* Other relevant information: 

### Config (Generated via diagnostics page)
<details><summary>Show Running Config</summary>

**Environment settings which are overridden:** DOMAIN, SIGNUPS_ALLOWED, ORG_CREATION_USERS, SHOW_PASSWORD_HINT, ADMIN_TOKEN, IP_HEADER, SMTP_HOST, SMTP_SECURITY, SMTP_PORT, SMTP_FROM, SMTP_FROM_NAME, SMTP_USERNAME, SMTP_PASSWORD

```
json
{
  "_duo_akey": null,
  "_enable_duo": true,
  "_enable_email_2fa": true,
  "_enable_smtp": true,
  "_enable_yubico": true,
  "_icon_service_csp": "",
  "_icon_service_url": "",
  "_ip_header_enabled": true,
  "_smtp_img_src": "cid:",
  "admin_ratelimit_max_burst": 3,
  "admin_ratelimit_seconds": 300,
  "admin_session_lifetime": 20,
  "admin_token": "***",
  "allowed_iframe_ancestors": "",
  "attachments_folder": "data/attachments",
  "auth_request_purge_schedule": "30 * * * * *",
  "authenticator_disable_time_drift": false,
  "data_folder": "data",
  "database_conn_init": "",
  "database_max_conns": 10,
  "database_timeout": 30,
  "database_url": "***************",
  "db_connection_retries": 15,
  "disable_2fa_remember": false,
  "disable_admin_token": false,
  "disable_icon_download": false,
  "domain": "*****://***********************",
  "domain_origin": "*****://***********************",
  "domain_path": "",
  "domain_set": true,
  "duo_host": null,
  "duo_ikey": null,
  "duo_skey": null,
  "email_2fa_auto_fallback": false,
  "email_2fa_enforce_on_verified_invite": false,
  "email_attempts_limit": 3,
  "email_change_allowed": true,
  "email_expiration_time": 600,
  "email_token_size": 6,
  "emergency_access_allowed": true,
  "emergency_notification_reminder_schedule": "0 3 * * * *",
  "emergency_request_timeout_schedule": "0 7 * * * *",
  "enable_db_wal": true,
  "enable_websocket": true,
  "event_cleanup_schedule": "0 10 0 * * *",
  "events_days_retain": null,
  "experimental_client_feature_flags": "fido2-vault-credentials",
  "extended_logging": true,
  "helo_name": null,
  "hibp_api_key": null,
  "icon_blacklist_non_global_ips": true,
  "icon_blacklist_regex": null,
  "icon_cache_folder": "data/icon_cache",
  "icon_cache_negttl": 259200,
  "icon_cache_ttl": 2592000,
  "icon_download_timeout": 10,
  "icon_redirect_code": 302,
  "icon_service": "internal",
  "incomplete_2fa_schedule": "30 * * * * *",
  "incomplete_2fa_time_limit": 3,
  "invitation_expiration_hours": 120,
  "invitation_org_name": "Vaultwarden",
  "invitations_allowed": true,
  "ip_header": "X-Real-IP",
  "job_poll_interval_ms": 30000,
  "log_file": "/data/bitwarden.log",
  "log_level": "Info",
  "log_timestamp_format": "%Y-%m-%d %H:%M:%S.%3f",
  "login_ratelimit_max_burst": 10,
  "login_ratelimit_seconds": 60,
  "org_attachment_limit": null,
  "org_creation_users": "*****************",
  "org_events_enabled": false,
  "org_groups_enabled": false,
  "password_hints_allowed": true,
  "password_iterations": 600000,
  "push_enabled": true,
  "push_identity_uri": "https://identity.bitwarden.com",
  "push_installation_id": "***",
  "push_installation_key": "***",
  "push_relay_uri": "https://push.bitwarden.com",
  "reload_templates": false,
  "require_device_email": false,
  "rsa_key_filename": "data/rsa_key",
  "send_purge_schedule": "0 5 * * * *",
  "sendmail_command": null,
  "sends_allowed": true,
  "sends_folder": "data/sends",
  "show_password_hint": false,
  "signups_allowed": true,
  "signups_domains_whitelist": "",
  "signups_verify": false,
  "signups_verify_resend_limit": 6,
  "signups_verify_resend_time": 3600,
  "smtp_accept_invalid_certs": false,
  "smtp_accept_invalid_hostnames": false,
  "smtp_auth_mechanism": null,
  "smtp_debug": false,
  "smtp_embed_images": true,
  "smtp_explicit_tls": null,
  "smtp_from": "*****************************",
  "smtp_from_name": "**************",
  "smtp_host": "********************",
  "smtp_password": "***",
  "smtp_port": 587,
  "smtp_security": "starttls",
  "smtp_ssl": null,
  "smtp_timeout": 15,
  "smtp_username": "*****************",
  "templates_folder": "data/templates",
  "tmp_folder": "data/tmp",
  "trash_auto_delete_days": null,
  "trash_purge_schedule": "0 5 0 * * *",
  "use_sendmail": false,
  "use_syslog": false,
  "user_attachment_limit": null,
  "user_send_limit": null,
  "web_vault_enabled": true,
  "web_vault_folder": "web-vault/",
  "yubico_client_id": null,
  "yubico_secret_key": null,
  "yubico_server": null
}
```

* Install method: [Bitwarden gCloud](https://github.com/dadatuputi/bitwarden_gcloud)

* Clients used: Windows App, Windows Chrome Extension

* Reverse proxy and version: Caddy (latest)

* Other relevant details:

### Steps to reproduce
[Vaulwarden gCloud](https://github.com/dadatuputi/bitwarden_gcloud)

### Expected behaviour
The apps open regularly and provide access to password list and profile. 

### Actual behaviour
The applications seemingly relock immediatly the app, but access through autofill seems to be available, access on android of the same user works flawlessly. 

### Troubleshooting data
![ScreenRecording2024-08-06at16 17 59-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/0babea48-6a93-49a8-89c8-84f15012ea1a)
https://github.com/user-attachments/assets/d2260f72-05b0-47f2-b020-2bb696cc3da5

**Comment by BlackDex** [comment]
I do not think we can fix this as it is a client issue.

But what do the logs say?
Is the date/time valid on the Windows Desktop?

**Reply by fthobe** [reply]
Hey @BlackDex 

1) the device seems to have a fried battery but the problem reproduces also with ntp freshly synched, I gonna test with a VM tomorrow if I can reproduce the error. 

2) what are the specific logs that could help me here?

**Reply by fthobe** [reply]
@BlackDex it's my first installation, could you give me some hints?

**Reply by BlackDex** [reply]
Just post the logs from Vaultwarden during these issues.
